.drone.star

```diff
@@ -0,0 +1,136 @@
+PLATFORMS = [
+    {"os": "linux", "arch": "amd64"},
+    {"os": "linux", "arch": "arm64"},
+    {"os": "darwin", "arch": "amd64"},
+]
+
+TRIGGER_CONDITION = {
+    "event": [
+        "push",
+        # "pull_request",
+        "tag",
+    ]
+}
+
+
+def get_docker_tags(repo, prefix, branch, commit):
+    tags = [
+        "latest",
+        "catalyst",
+        branch.replace("/", "-"),
+        "$DRONE_BUILD_CREATED",
+        commit,
+        commit[:8],
+    ]
+    return ["%s:%s-%s" % (repo, prefix, tag) for tag in tags]
```

Comment:
I'm not sure I like `latest` getting pushed for every build; while technically true I wouldn't except `docker run livepeer/catalyst` to give me whatever build on a branch has gone out first.

Reply:
yes. i wanted to experiment with the build context before updating this. the tags generated would (should?) be:
- branch_name_in_snake_case
- commit sha
- short commit sha
- latest only if branch-name == catalyst
- some CI unique identifier